flecs_ecs/examples/flecs/prefabs/prefab_override.rs

```diff
@@ -63,7 +63,7 @@ fn main() {
 
     // Even though Attack was not automatically overridden, we can always
     // override it manually afterwards by adding it:
-    inst.override_type::<Attack>();
+    inst.add::<Attack>();
```

Comment:
this will panic because `Attack` is not default. 

This might be why I had this function. Your only other solution is doing set, but then you're not inheriting / copying the values of Attack.

Reply:
Not sure what you want changed here.